src/main/resources/hudson/plugins/synergy/SynergySCM/global.jelly

```diff
@@ -1,17 +1,17 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 	<f:section title="Synergy">
-		<f:entry title="ccm executable" help="/plugin/synergy/ccmexe.html">
+		<f:entry title="ccm executable">
 			<f:textbox name="synergy.ccmExe" value="${descriptor.ccmExe}"
 				checkUrl="'${rootURL}/scm/SynergySCM/ccmExeCheck?value='+escape(this.value)" />

```

Comment:
`checkUrl` can be removed if using the conventional naming pattern on the `Descriptor`: `doCheckCcmExe`.